scss/_patterns_pull-quotes.scss

```diff
@@ -27,7 +27,9 @@
 
     // this needs to be nested to increase the specificity
     .p-pull-quote__citation {
-      font-style: italic;
+      @extend %vf-heading-6;
```

Comment:
Do we wan't the same h6 size for all the variants?

This part of SCSS file is defining common shared styles of all quotes. Specific sizes for all variants are defined below:
line 79 defines size of default variant (as h4)
line 96 defines small variant as default text size
line 114 defines large variant as h3

I don't know why this shared definition of h6 wins in specificity and overrides all of the specific styles defined later.

If we really want the same size for all variants, we should remove individual size definitions below.